source/_patterns/01-molecules/navigation/main-menu.yaml

```diff
@@ -15,14 +15,9 @@ properties:
           url:
             type: string
             minLength: 1
-          hasBorderBottom:
-            type: boolean
-            enum:
-              - true
-          isHiddenMainMenuItemForWideScreen:
-            type: boolean
-            enum:
-              - true
+          classes:
```

Comment:
I will create a followup ticket for instances of link to use the `non-pattern-schema/link.yaml` schema and we can added classes to it.